{"name": "test_incorrect[incorrect_data2]", "status": "passed", "attachments": [{"name": "log", "source": "d811557d-d45d-452f-8cf5-18022142047a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '1303845892', 'password': 'nmb_python', 'check': '用户不存在'}"}], "start": 1594727249202, "stop": 1594727251976, "uuid": "da991aae-7453-4860-8a31-1a00f806dfa0", "historyId": "9a23aa568774cb1eb1113c9dc5db5449", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "10824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}